src/server.conf.in

```diff
@@ -101,13 +101,13 @@ serverRealm = local
 ; extra-* template slots. If the extra folder has a sub folder "static", the
 ; resources in this static folder will be available as /extra/static/filename
 ; relative to your servers base URL.
-;extra = /usr/share/spreed-webrtc-server/extra
+extra = /usr/share/spreed-webrtc-server/extra
```

Comment:
Where does this path come from? I do not see it in the pr and it probably needs to be different when run in a snap.